### PR TITLE
Bugfix MTE-1783 [v120] Update account branding

### PR DESCRIPTION
**Pull request by clarmso**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1783)

## :bulb: Description
Firefox Account is now known as Mozilla Account. The branding on the sign in page has been updated. This PR updates the static text to be expected when the login page is loaded.

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [X] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Tests/XCUITests/IntegrationTests.swift [hunk 1]
```diff
@@ -48,14 +48,15 @@ class IntegrationTests: BaseTestCase {
             return true
         }
         app.swipeDown()
+        sleep(5)
```
Comment (dragosb01) [inline Tests/XCUITests/IntegrationTests.swift:51]: Is this sleep here necessary? Cant a wait function be used here instead? Just asking, if this is the only solution then i am ok with it?

Comment (clarmso) [inline Tests/XCUITests/IntegrationTests.swift:51]: Unrelated to this PR's subject: Clicking "Allow" from the notification before proceeding to the following step seem to stablize the tessts.

Comment (clarmso) [reply Tests/XCUITests/IntegrationTests.swift:51]: The lack of wait may be exposing an issue on Firefox itself.

**Review by dragosb01 — CHANGES_REQUESTED** [review]

File: Tests/XCUITests/IntegrationTests.swift (hunk 1 above)
Comment (clarmso) [reply Tests/XCUITests/IntegrationTests.swift]: Let me remove it for now and open a separate PR to address the issue.

**Review by isabelrios — APPROVED** [review]

**Pull request merged**